src/components/table/Table.vue

```diff
@@ -66,13 +66,13 @@
                             :key="index"
                             :class="[rowClass(row, index), {
                                 'is-selected': row === selected,
-                                'is-checked': isRowChecked(row)
+                                'is-checked': isRowChecked(row),
                             }]"
-                            @click="selectRow(row)"
+                            @click="detailed && !chevron ? toggleOnclick(row) : selectRow(row)"
                             @dblclick="$emit('dblclick', row)">
```

Comment:
Remove it. I want to let free to toggle detail row using directly the method `toggleDetails(row)`, so a developer could be open the detail as he wants.

Reply:
i'm not sure i 100% understand you.
soon as i restore it,
the `toggleDetails(row)` wont be called if you hide the chevron column & icon, so the table will be just a normal table.

Reply:
Yes a normal table and the developer choose how toggle detail section. For example click all row or button or custom icon

Reply:
got it.
i will make the changes and commit soon